scripts/build-docker.sh

```diff
@@ -24,14 +24,17 @@ tar \
   -czf medplum-server.tar.gz \
   package.json \
   package-lock.json \
+  node_modules \
```

Comment:
I'm not sure this works with multi arch builds?

Reply:
Most of the packages include the native bindings for all architectures. Not a guarantee but there's a chance it works